### PR TITLE
Add angular-imgcache.js@0.1.0 w/ git auto-update

### LINKED ISSUE #7742: [Request] add angular-imgcache.js
**Library name:** angular-imgcache.js
**Git repository url:** https://github.com/jBenes/angular-imgcache.js
**npm package url(optional):**
**License(s):** MIT
**Official homepage:**
**Wanna say something? Leave message 
# 

Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
close #7742, cc @jBenes, using web-minify-helper tool for minifing

Pull request for issue: #7742
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/jBenes/angular-imgcache.js
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference:MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 10
   - Count of stars: 92
   - Count of forks: 48
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/angular-imgcache.js/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "angular-imgcache.js",
+  "version": "0.1.0",
+  "description": "Simple imgcache.js wrapper for AngularJS, can be user with Ionic/Cordova/Phonegap.",
```
Comment (extend1994) [inline ajax/libs/angular-imgcache.js/package.json]: Is `user` a typo here? Suppose it's `used`?

Comment (sashberd) [reply ajax/libs/angular-imgcache.js/package.json]: @extend1994 I used author`s description, but I thing you are right, I will update it

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by extend1994** [comment]
I just pushed a branch with wrong version. Will fix it in a minute.

**Comment by PeterDaveHello** [comment]
@sufuf3 don't bother too much about the map file in this case, and I don't think you should do so many changes for @sashberd, why not just ask him to do it so he'll know what should be done in future?

**Comment by sufuf3** [comment]
Sorry that I will do it in future.

**Pull request merged**